pkg/proxy/iptables/proxier.go

```diff
@@ -1396,6 +1398,20 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	// fetch number of packets dropped by iptables which were marked INVALID by conntrack.
+	if !proxier.conntrackTCPLiberal {
+		proxier.iptablesData.Reset()
+		if err := proxier.iptables.SaveInto(utiliptables.TableFilter, proxier.iptablesData, utiliptables.SaveCounters); err == nil {
+			pCounter, bCounter, err := utiliptables.GetCountersForRule(proxier.iptablesData.Bytes(), conntrackStateInvalidDropRule)
+			if err != nil {
+				klog.ErrorS(err, "Failed to get counter for conntrack state invalid drop rule")
+			} else {
+				klog.InfoS("Fetched counter for conntrack state invalid drop rule", "packets", pCounter, "bytes", bCounter)
```

Comment:
probably don't need to log this... you can get the info from the metric, so it doesn't need to be in the logs. (Or at least, not at `V(1)` anyway)